CPAC/registration/registration.py

```diff
@@ -1960,6 +1977,227 @@ def register_ANTs_EPI_to_template(wf, cfg, strat_pool, pipe_num, opt=None):
         wf.connect(node, out, ants, 'inputspec.reference_mask')
 
     return (wf, outputs)
+    
+    
+def apply_transform_anat_to_template(wf, cfg, strat_pool, pipe_num, opt=None):
+    '''
+    {"name": "apply_transform_anat_to_template",
+     "config": ["registration_workflows", "anatomical_registration"],
+     "switch": ["run"],
+     "option_key": ["apply_transform", "using"],
+     "option_val": ["default", "ANTS", "FSL"],
+     "inputs": ["desc-restore-brain_T1w", 
+                ["desc-brain_T1w", "space-longitudinal_desc-brain_T1w"],
+                ["desc-restore_T1w", "desc-preproc_T1w", "desc-reorient_T1w", "T1w"],
+                "space-T1w_desc-brain_mask",
+                "T1w_template",
+                "from-T1w_to-template_mode-image_xfm"],
+     "outputs": ["space-template_desc-brain_T1w",
+                 "space-template_desc-T1w_mask",
+                 "from-T1w_to-template_mode-image_xfm"]}
```

Comment:
Does the T1w to template transform get modified in this node block?

Reply:
It does in abcd-options pipeline (ANTs for registration, FSL for applywarp) because we need to transform ANTs warp to FSL format, but it doesn't change in default pipeline which uses consistent tool for registration and applywarp.

Reply:
Okay good. Wouldn't change anything then.